pebble/pathapp/src/pathapp.c

```diff
@@ -0,0 +1,285 @@
+#include "pebble.h"
```

Comment:
Don't see an easy way to perform integration tests for pebble in C or mock the message-passing or Layer-based UI generation yet. I can look into it. Is it fine if I make an issue now and address this in next PR?